src/components/manager/runs/RunLastContent.js

```diff
@@ -16,47 +17,116 @@ const COLUMNS_LAST_CONTENT = [
         title: "File Names",
         dataIndex: "fileNames",
         key: "fileNames",
-        render: (fileNames) => (
-            <>
-                {fileNames.map((fileName, index) => (
-                    <div key={index}>{fileName}</div>
-                ))}
-            </>
-        ),
+        render: (fileNames, record) => <FileNamesCell fileNames={fileNames} dataType={record.dataType}/>,
     },
 ];
 
+function FileNamesCell({fileNames, dataType}) {
+    const [isModalVisible, setIsModalVisible] = useState(false);
+
+    const isTruncated = fileNames.length > 4;
+    const truncatedFileNames = isTruncated
+        ? [...fileNames.slice(0, 2), <Icon type="more" key="more-icon"/>, ...fileNames.slice(-2)]
+        : fileNames;
```

Comment:
Sure, If there are more than 4 file names (fileNames.length > 4), it shows a truncated version (first 2 and last 2 names, with an icon in between). Otherwise, it shows all file names. This is done to improve representation when several files are ingested.

Reply:
agree with code comment, otherwise lgtm

Reply:
Yes, it is more clear.. !